ledger/accountdb.go

```diff
@@ -2130,11 +2130,16 @@ func performTxTailTableMigration(ctx context.Context, tx *sql.Tx, blockDb db.Acc
 		return nil
 	}
 
+	latest, err := accountsRound(tx)
```

Comment:
please rename to `dbRound` for consistency

Reply:
done